export-helpers/background.js

```diff
@@ -0,0 +1,30 @@
+function onExecuted() {
+  if (chrome.runtime.lastError) {
+    console.log(chrome.runtime.lastError);
+  }
+}
+
+/*
+Execute content script in the active tab.
+*/
+function loadContentScript() {
+  chrome.tabs.executeScript({
```

Comment:
If the example is meant to be Firefox specific we could change all the `chrome.*` API calls into `browser.*` API calls
(and given that it demonstrates a Firefox specific feature, it looks like it could be, the only part that could be cross-browser is the suggested "forward an object received through the messaging API to the page in a custom event" scenario).